<launch>

	
  <group ns="turtlesim1">
    <node pkg="turtlesim" name="turtle1" type="turtlesim_node"/>
  </group>

  <group ns="turtlesim2">
    <node pkg="turtlesim" name="turtle1" type="turtlesim_node"/>
  </group>
  

  
  	

</launch>
